pre_push.py

```diff
@@ -15,36 +15,37 @@ def do_process(*args):
 
     Exit if command is not found.
     """
-    print('Running: {}'.format(' '.join(args)))
+    print("Running: {}".format(" ".join(args)))
     try:
-        check_call(args)
+        check_call(args, shell=shell)
     except CalledProcessError:
-        print('\nFailed: {}'.format(' '.join(args)))
+        print("\nFailed: {}".format(" ".join(args)))
         return False
     except Exception as exc:
-        sys.stderr.write(str(exc) + '\n')
+        sys.stderr.write(str(exc) + "\n")
         sys.exit(1)
     return True
 
 
 def main():
     """Entry point to pre_push.py."""
     success = True
-    success &= do_process('flake8', '--exclude=.eggs,docs,.tox')
-    success &= do_process('pydocstyle', 'praw')
-    success &= do_process('pylint', '--rcfile=.pylintrc', 'praw')
+    success &= do_process(["black -l 79 *.py docs praw tests"], shell=True)
+    success &= do_process(["flake8", "--exclude=.eggs,docs,.tox"])
+    success &= do_process(["pydocstyle", "praw"])
+    # success &= do_process(["pylint", "--rcfile=.pylintrc", "praw"])
```

Comment:
Why is pylint commented out?

Reply:
I didn't satisfactorily resolve the issues it was complaining about after adding in black for source code formatting. So I've commented it out.